test/e2e/autotls/auto_tls_test.go

```diff
@@ -124,6 +125,58 @@ func TestPerNamespaceCertLocalCA(t *testing.T) {
 	testingress.RuntimeRequest(t, httpsClient, "https://"+objects.Service.Status.URL.Host)
 }
 
+func TestRouteTagLocalCA(t *testing.T) {
```

Comment:
This file has been significantly changed by my PR https://github.com/knative/serving/pull/6651 (sorry about that). so you may have to do the rebase.

In addition, can we add the test logic into the existing function [testAutoTLS](https://github.com/knative/serving/blob/1b57732f01a5735441cbf497aea2b80992d69689/test/e2e/autotls/auto_tls_test.go#L64) instead of having a separate test? I feel some setups seems to be repeated in this test.

Reply:
No problem, I'll get it rebased